dev-packages/e2e-tests/registrySetup.ts

```diff
@@ -85,8 +85,15 @@ export function registrySetup(): void {
       },
     );
 
-    if (publishImageContainerRunProcess.status !== 0) {
-      throw new Error('Publish Image Container failed.');
+    const statusCode = publishImageContainerRunProcess.status;
+
+    if (statusCode !== 0) {
+      if (statusCode === 137) {
```

Comment:
l: I am never a big fan of having such nested if clauses :D Instead, what about doing something like this:

```js
const message = statusCode === 137 ? '....' : `Publish Image Container failed with exit code ${statusCode}`;
throw new Error(message);
```